### PR TITLE
Update Nim to search for Nimble packages in "pkg2"

### PR DESCRIPTION
**Pull request by bobeff**
Update the compiler to search for nimble packages in the "pkg2" subdirectory instead of in "pkg". This should be merged when the new Nimble is released.

### CODE REVIEW

File: tools/niminst/niminst.nim [hunk 1]
```diff
@@ -63,6 +63,7 @@ type
     nimArgs: string
     debOpts: TDebOptions
     nimblePkgName: string
+    checksum: string
```
Comment (Araq) [inline tools/niminst/niminst.nim]: This is never set?

Comment (bobeff) [reply tools/niminst/niminst.nim]: I think that it is accessed in `install.nimf` and `deinstall.nimf` but I'm not 100% sure that it is the same variable.

Comment (bobeff) [reply tools/niminst/niminst.nim]: I renamed it to `nimblePkgChecksum` and I set it in the `parseIniFile` procedure the same way as the `nimblePkgName` field.

Comment (Araq) [reply tools/niminst/niminst.nim]: > I think that it is accessed in install.nimf and deinstall.nimf but I'm not 100% sure that it is the same variable.

Well niminst wouldn't compile if you used an undeclared variable.

**Pull request closed without merging**